tests/e2e/specs/stories-editor/media-file-size-warnings.test.js

```diff
@@ -92,5 +88,24 @@ describe( 'Media File Size Warnings', () => {
 			// It should be possible to insert the uploaded video, as this should not disable the 'Select' button.
 			await expect( page ).toClick( SELECT_BUTTON );
 		} );
+
+		it( 'should not display a warning in the Media Library for a correct video file size', async () => {
+			// Using the regular inserter.
+			await insertBlock( 'Video' );
+
+			// Click the media library button.
+			await page.waitForSelector( '.editor-media-placeholder__media-library-button' );
+			await page.click( '.editor-media-placeholder__media-library-button' );
+			await uploadMedia( CORRECT_VIDEO );
+
+			// The warning Notice component should not appear.
+			await expect( page ).not.toMatchElement( NOTICE_SELECTOR );
+
+			// The warning notice text should not appear.
+			await expect( page ).not.toMatch( EXPECTED_NOTICE_TEXT );
```

Comment:
Maybe this should also search for the notice text inside the Media Library, not the page. 

But since this is verifying that it doesn't appear, I thought it'd be good to ensure it doesn't appear anywhere.

Reply:
Makes sense to keep it like this 👍